cmd/cluster-kube-controller-manager-operator/render/render.go

```diff
@@ -166,6 +166,14 @@ func (r *renderOpts) Run() error {
 	if renderConfig.Assets, err = assets.LoadFilesRecursively(r.assetInputDir, skipSchedulerPredicate); err != nil {
 		return fmt.Errorf("failed loading assets from %q: %v", r.assetInputDir, err)
 	}
+
+	// load kubeconfig for bootkube phase 2
+	if kubeConfig, err := r.readBootstrapSecretsKubeconfig(); err != nil {
```

Comment:
isn't this automatically included in the upper LoadFilesRecursively?

Reply:
yes...

Reply:
actually not... bootstrap-secrets does not seem to be included

Reply:
oh, it's a different dir?